lib/saml-form.ts

```diff
@@ -6,17 +6,31 @@ import * as escape from 'escape-html'
 
 export function createSamlForm (ssoLocation: string, samlRequest: string) {
   return `
-    <form method='post' action='${escape(ssoLocation)}'>
-      <h1>Send SAML Authn request to hub</h1>
+    <form class='passport-verify-saml-form' method='post' action='${escape(ssoLocation)}'>
+      <h1>Continue to next step</h1>
+      <p>Because Javascript is not enabled on your browser, you must press the continue button</p>
       <input type='hidden' name='SAMLRequest' value='${escape(samlRequest)}'/>
       <input type='hidden' name='relayState' value=''/>
-      <button>Submit</button>
+      <button class='passport-verify-button'>Continue</button>
     </form>
     <script>
       var form = document.forms[0]
       form.setAttribute('style', 'display: none;')
       window.setTimeout(function () { form.removeAttribute('style') }, 5000)
       form.submit()
     </script>
+    <style type='text/css'>
+      .passport-verify-saml-form {
+        font-family: Arial, sans-serif;
+      }
+      .passport-verify-button {
+        background-color: #00692f;
+        color: #fff;
+        padding: 10px;
+        font-size: 1em;
+        border: none;
+        box-shadow: 0 2px 0 #003618;
```

Comment:
Can we stick `cursor: pointer` in this too? There's some controversy over that:

https://ux.stackexchange.com/questions/3788/default-cursor-on-mouse-over-of-a-button-is-not-a-hand-pointer

But gov.uk elements puts the pointer on it (amongst some other things).

GOV.UK elements also has a few other bits and bobs for .button - I think:

```
.button {
  background-color: #00823b;
  position: relative;
  display: -moz-inline-stack;
  display: inline-block;
  padding: .526315em .789473em .263157em;
  border: none;
  -webkit-border-radius: 0;
  -moz-border-radius: 0;
  border-radius: 0;
  outline: 1px solid transparent;
  outline-offset: -1px;
  -webkit-appearance: none;
  -webkit-box-shadow: 0 2px 0 #003618;
  -moz-box-shadow: 0 2px 0 #003618;
  box-shadow: 0 2px 0 #003618;
  font-size: 1em;
  line-height: 1.25;
  text-decoration: none;
  -webkit-font-smoothing: antialiased;
  cursor: pointer;
  color: #fff;
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
  vertical-align: top; }
  .button:visited {
    background-color: #00823b; }
  .button:hover, .button:focus {
    background-color: #00692f; }
  .button:active {
    top: 2px;
    -webkit-box-shadow: 0 0 0 #00823b;
    -moz-box-shadow: 0 0 0 #00823b;
    box-shadow: 0 0 0 #00823b; }
  .button.disabled, .button[disabled="disabled"], .button[disabled] {
    zoom: 1;
    filter: alpha(opacity=50);
    opacity: 0.5; }
    .button.disabled:hover, .button[disabled="disabled"]:hover, .button[disabled]:hover {
      cursor: default;
      background-color: #00823b; }
    .button.disabled:active, .button[disabled="disabled"]:active, .button[disabled]:active {
      top: 0;
      -webkit-box-shadow: 0 2px 0 #003618;
      -moz-box-shadow: 0 2px 0 #003618;
      box-shadow: 0 2px 0 #003618; }
  .button:link, .button:link:focus, .button:hover, .button:focus, .button:visited {
    color: #fff; }
  .button:before {
    content: "";
    height: 110%;
    width: 100%;
    display: block;
    background: transparent;
    position: absolute;
    top: 0;
    left: 0; }
  .button:active:before {
    top: -10%;
    height: 120%; }
  @media (max-width: 640px) {
    .button {
      width: 100%;
      text-align: center; } }

.button::-moz-focus-inner {
  border: 0;
  padding: 0; }

.button:focus {
  outline: 3px solid #ffbf47; }

.button[disabled="disabled"] {
  background: #00823b; }

.button[disabled="disabled"]:focus {
  outline: none; }
```

... not all of which is necessary, but we could maybe get a slightly better look and feel.